README.md

```diff
@@ -106,15 +106,22 @@ This should produce spans that can be captured in the Lightstep Explorer.
 |service_name|OTEL_SERVICE_NAME|y|-|
 |service_version|LS_SERVICE_VERSION|n|`None`|
 |access_token|LS_ACCESS_TOKEN|n|`None`|
+|metrics_enabled|LS_METRICS_ENABLED|n|`False`|
```

Comment:
is this variable common across launchers and if not will it be?

Reply:
We do in Java at least